python/ray/serve/utils.py

```diff
@@ -342,3 +342,37 @@ def get_node_id_for_actor(actor_handle):
     """Given an actor handle, return the node id it's placed on."""
 
     return ray.actors()[actor_handle._actor_id.hex()]["Address"]["NodeID"]
+
+
+def import_class(full_path: str):
+    """Given a full import path to a class name, return the imported class.
+
+    For example, the following are equivalent:
+        MyClass = import_class("module.submodule.MyClass")
+        from module.submodule import MyClass
+
+    Returns:
+        Imported class
+    """
+
+    last_period_idx = full_path.rfind(".")
```

Comment:
Can we use https://docs.python.org/3/library/importlib.html#importlib.import_module instead? It handles more cases and we don't need to do manual parsing.

Reply:
We'll still need to do a little parsing for the class, but sure this seems slightly better